### PR TITLE
feat: Ability to show something in the search part of the bar

### PR DESCRIPTION
**Pull request by ptbrowne**
With this, we will be able to show a custom search bar in the bar.

Should Drive use this capacity to show its search bar (this way the bar would not carry the code that is at the moment very specific to drive) ?

### CODE REVIEW

File: test/lib/__snapshots__/api.spec.jsx.snap [hunk 1]
```diff
@@ -25,6 +25,8 @@ exports[`api spec should set content of the store via declarative api 3`] = `
 </div>
 `;
 
+exports[`api spec should set content of the store via declarative api 4`] = `<div />`;
```
Comment (Crash--) [inline test/lib/__snapshots__/api.spec.jsx.snap:28]: what does that test ? ^^

**Review by Crash-- — APPROVED** [review]

**Comment by Crash--** [comment]
> Should Drive use this capacity to show its search bar (this way the bar would not carry the code that is at the moment very specific to drive) ?

At first, we thought to be able to provide a generic way to search in all your cozy. I never read the code, just took a look to write this comment but it seems not specific to drive even if it's only used in Drive ATM. 

Does this removal will have a "big" impact in the bundle size? 

Let's discuss that point next week

File: test/lib/__snapshots__/api.spec.jsx.snap (hunk 1 above)
Comment (ptbrowne) [reply test/lib/__snapshots__/api.spec.jsx.snap]: I had forgotten to put a content for the search. Fixed in https://github.com/cozy/cozy-bar/pull/706/commits/789de16c4f2c8064a9a79562945252a6a9a7571d

**Comment by ptbrowne** [comment]
I've added the point to the card in Trello.

**Pull request merged**